volume-group-operator/controllers/persistentvolumeclaim/persistentvolumeclaim_controller.go

```diff
@@ -19,6 +19,8 @@ package persistentvolumeclaim
 import (
 	"context"
 
+	csiv1 "github.com/IBM/volume-group-operator/api/v1"
```

Comment:
please change `PersistentVolumeClaimWatcher` to `PersistentVolumeClaimReconciler`

Reply:
[done](https://github.com/IBM/ibm-block-csi-operator/pull/305)